constants.py

```diff
@@ -25,6 +25,7 @@
 
 # Resource types.
 RESOURCE_TYPE_NETWORK_POLICY = "NetworkPolicy"
+RESOURCE_TYPE_EGRESS_POLICY = "Egresspolicy"
```

Comment:
I think we should try to make the naming of this TPR less egress-centric so that we can use this TPR to implement more features than just egress down the road.

What about dropping the egress and calling this `Policy` (or `CalicoPolicy`)?  Any idea how TPRs behave when they match the same name as a k8s resource? i.e. what would happen if we called this `NetworkPolicy`?